app/connectors/EventReportConnector.scala

```diff
@@ -81,12 +81,12 @@ class EventReportConnector @Inject()(
     }
   }
 
-  def getEvent(pstr: String, startDate: String, version: String, eventType: Option[EventType])
+  def getEvent(pstr: String, startDate: String, version: Int, eventType: Option[EventType])
               (implicit headerCarrier: HeaderCarrier, ec: ExecutionContext): Future[Option[JsValue]] = {
     val headers = integrationFrameworkHeader ++
       Seq(
         "reportStartDate" -> startDate,
-        "reportVersionNumber" -> version
+        "reportVersionNumber" -> ("00" + version.toString).takeRight(3)
```

Comment:
Not that I anticipate this to be a real issue, but how would we deal with very large version numbers with this interpretation? How high do version numbers typically go? I guess this would only be a problem for version 1000+...